Command/ActivateUserCommand.php

```diff
@@ -56,13 +56,15 @@ protected function configure()
     /**
      * {@inheritdoc}
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $username = $input->getArgument('username');
 
         $this->userManipulator->activate($username);
 
         $output->writeln(sprintf('User "%s" has been activated.', $username));
+
+        return 0;
```

Comment:
these return statements should be part of 2.2.0, to avoid deprecation warnings there (returning an integer is supported since Symfony 2.0.0 anyway so that won't break anything).

Reply:
https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2956